gui-apps/organicmaps/organicmaps-9999.ebuild

```diff
@@ -29,27 +27,35 @@ REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 # depend on sys-libs/zlib[minizip] when it is not pulled in as subproject anymore
 RDEPEND="
+	>dev-cpp/fast_double_parser-0.7.0
 	dev-cpp/gflags
 	dev-db/sqlite
 	dev-lang/python
 	dev-libs/boost
+	dev-libs/expat
 	dev-libs/icu
 	dev-libs/jansson
-	dev-libs/utfcpp
+	>=dev-libs/utfcpp-4
+	dev-qt/qtbase:6[gui,network,opengl,widgets]
 	dev-qt/qtpositioning:6
 	dev-qt/qtsvg:6
-	dev-util/vulkan-headers
+	>=dev-util/vulkan-headers-1.3.280
```

Comment:
This is a dependency of `dev-qt/qtbase`

Reply:
In qtbase no version in pinned. Does OM link directly against vulkan-headers? In this case, add is as dependency, too, since qtbase can drop its dependency some time while OM will keep it.

Reply:
Ok added this line, with version.